### ISSUE TITLE
schema[playbook]: $[1].order 'inventory' is not one of ['default', 'sorted', 'reverse_sorted', 'reverse_inventory', 'shuffle']

### ISSUE THREAD

**Issue by RayJin2000** [ansible/ansible-lint#4163]
##### Summary
If you use 'inventory' as playbook order value you get the following error:

```paste below
schema[playbook]: $[1].order 'inventory' is not one of ['default', 'sorted', 'reverse_sorted', 'reverse_inventory', 'shuffle']
```
See: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_strategies.html#ordering-execution-based-on-inventory

Workaround: use default instead of inventory

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 24.2.3 using ansible-core:2.16.6 ansible-compat:4.1.11 ruamel-yaml:0.18.6 ruamel-yaml-clib:0.2.7
```

##### STEPS TO REPRODUCE

```yaml (paste below)
- name: Test
  hosts: localhost
  order: inventory
  gather_facts: false
```

```console (paste below)
 > ansible-lint playbooks/test.yml --fix
WARNING  Listing 1 violation(s) that are fatal
schema[playbook]: $[0].order 'inventory' is not one of ['default', 'sorted', 'reverse_sorted', 'reverse_inventory', 'shuffle']
playbooks/test.yml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``schema[playbook]``,
``schema[tasks]``.

This rule is not skippable and stops further processing of the file.

If incorrect schema was picked, you might want to either:

* move the file to standard location, so its file is detected correctly.
* use ``kinds:`` option in linter config to help it pick correct file type.


Read documentation for instructions on how to ignore specific rule violations.

Modified 1 files.
               Rule Violation Summary                
 count tag              profile rule associated tags 
     1 schema[playbook] basic   core                 

Failed: 1 failure(s), 0 warning(s) on 1 files. Last profile that met the validation criteria was 'min'.

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
